src/Core/Content/Cms/DataResolver/Element/AbstractCmsElementResolver.php

```diff
@@ -41,7 +41,11 @@ protected function resolveEntityValue(?Entity $entity, string $path)
             }
 
             try {
-                $value = $value->get($part);
+                if (is_array($value)) {
+                    $value = $value[$part];
+                } else {
+                    $value = $value->get($part);
+                }
```

Comment:
You can use the ternary operator here:
```suggestion
                $value = is_array($value) ? $value[$part] : $value->get($part);
```

Reply:
please implement that sugestion

Reply:
@NiklasLimberg is already...